tests/cli_test.ts

```diff
@@ -286,6 +288,8 @@ Deno.test({
       await lines.cancel();
       serverProcess.kill("SIGTERM");
       serverProcess.close();
+
+      await Deno.remove(tmpDirName, { recursive: true });
```

Comment:
Please do this outside of the test step (in the top level test itself). Just move the code 1 line down.